examples/ws/BPEL/AirportInfoServices/nbproject/private/private.properties

```diff
@@ -0,0 +1,4 @@
+com.sun.jbi.ui.devtool.jbi.component.conf.file=ComponentInformation.xml
+com.sun.jbi.ui.devtool.jbi.deployment.conf.file=default.xml
+module.install.dir=/encs/pkg/openesb-3.0.5/root/OE-Studio/netbeans/soa/modules
+netbeans.user=/groups/j/ji_soen487_4/aa
```

Comment:
- [ ] I don't think private user-specific config should be in the repo with hardcoded paths. Can you check if this file is not included and you run it from another place like laptop under a different user, without this file present, will it regenerate it properly to your current location?